config/application.rb

```diff
@@ -55,21 +55,27 @@ class Application < Rails::Application
     end
 
     initializer "spree.environment", before: :load_config_initializers do |app|
-      app.config.spree = Spree::Core::Environment.new
-      Spree::Config = app.config.spree.preferences # legacy access
+      Rails.application.reloader.to_prepare do
+        app.config.spree = Spree::Core::Environment.new
+        Spree::Config = app.config.spree.preferences # legacy access
+      end
     end
 
     initializer "spree.register.payment_methods" do |app|
-      app.config.spree.payment_methods = [
-        Spree::Gateway::Bogus,
-        Spree::Gateway::BogusSimple,
-        Spree::PaymentMethod::Check
-      ]
+      Rails.application.reloader.to_prepare do
+        app.config.spree.payment_methods = [
+          Spree::Gateway::Bogus,
+          Spree::Gateway::BogusSimple,
+          Spree::PaymentMethod::Check
```

Comment:
The modern way is to convert these to strings and reference the classes later in the code.

Reply:
That sounds good :+1: